README.md

```diff
@@ -1,7 +1,7 @@
 # framepay-react
 
 [![npm](https://img.shields.io/npm/v/@rebilly/framepay-react.svg)](https://www.npmjs.com/package/@rebilly/framepay-react)
-[![Build Status](https://travis-ci.org/Rebilly/framepay-react.svg?branch=master)](https://travis-ci.org/Rebilly/framepay-react)  
+[![Build Status](https://github.com/Rebilly/framepay-react/workflows/Build & Test/badge.svg)](https://github.com/Rebilly/framepay-react/actions)  
```

Comment:
```suggestion
[![Build Status](https://github.com/Rebilly/framepay-react/workflows/Build%20&%20Test/badge.svg)](https://github.com/Rebilly/framepay-react/actions)
```

Reply:
It does not preview correctly without the spaces converted, check out the preview: https://github.com/Rebilly/framepay-react/blob/50b41ec5c76b443226e200ac522c9819c277e9e1/README.md

Reply:
Yes, you are right, I copied encoded URL from browser, not sure why it pasted decoded, and I have not noticed. Thanks.